modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20397,6 +20398,7 @@ to-navigate=to navigate
 to-save-this-form-all-required-fields-of-the-selected-object-need-to-be-mapped=To save this form, all required fields of the selected object need to be mapped.
 to-select=to select
 to-send=To Send
+to-set-a-workflow-go-to-process-builder-configuration=To set a workflow, go to Applications > Process Builder > Configuration. Refresh this page to see the changes.
```

Comment:
in here, considering the pattern from the key bellow and thinking about Brian's approval, i believe we can add the "Applications" step to the key, like: `to-set-a-workflow-go-to-applications-process-builder-configuration=To set a workflow, go to Applications > Process Builder > Configuration. Refresh this page to see the changes.`

But feel free to do it as you see fit, it's just a personal preference!